modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-container-test/src/testIntegration/resources/META-INF/resources/runtime_portlet.jsp

```diff
@@ -16,8 +16,13 @@
 
 <%@ taglib uri="http://liferay.com/tld/portlet" prefix="liferay-portlet" %>
 
-<%@ page import="com.liferay.portal.kernel.util.ParamUtil" %>
+<%@ page import="com.liferay.portal.kernel.util.ParamUtil" %><%@
+page import="com.liferay.portal.kernel.util.Validator" %>
+
+<%
+String portletName = ParamUtil.getString(request, "testRuntimePortletId");
+%>
 
 <liferay-portlet:runtime
-	portletName='<%= ParamUtil.getString(request, "testRuntimePortletId") %>'
+	portletName='<%= Validator.isBlank(portletName) ? (String)request.getAttribute("testRuntimePortletId") : portletName %>'
```

Comment:
@yuhai Is it a convention, or common practice to use 1) `Validator.isBlank` instead of `Validator.isNull` and 2) `condition ? A : B` in JSP in Liferay?

Reply:
> @yuhai Is it a convention, or common practice to use 1) Validator.isBlank instead of Validator.isNull and 2) condition ? A : B in JSP in Liferay?

@dantewang, 
1), I think this is common use Validator.isBlank and Validator.isNull is jsp. You may search Validator.isBlank and Validator.isNull in jsp, you can see lots of places use them. ParamUtil.getString() return blank, so I used isBlank.

2) yes, lots of places use this way in jsp.

Reply:
@yuhai OK, as long as it's commonly used I can accept it.

@tinatian Passed my review.